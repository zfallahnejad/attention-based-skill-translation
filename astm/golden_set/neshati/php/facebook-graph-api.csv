1343690,facebook-graph-api
1427878,facebook-graph-api
757508,facebook-graph-api
21062,facebook-graph-api
1603357,facebook-graph-api
337227,facebook-graph-api
52087,facebook-graph-api
920568,facebook-graph-api
875362,facebook-graph-api
558021,facebook-graph-api
155033,facebook-graph-api
265570,facebook-graph-api
912623,facebook-graph-api
1515819,facebook-graph-api
942852,facebook-graph-api
597384,facebook-graph-api
350858,facebook-graph-api
1015483,facebook-graph-api
702469,facebook-graph-api
718224,facebook-graph-api
