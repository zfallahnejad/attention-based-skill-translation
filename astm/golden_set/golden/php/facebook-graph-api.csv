1343690,99,71,0.5823529411764706
1427878,42,42,0.5
757508,41,55,0.4270833333333333
21062,39,28,0.582089552238806
1603357,37,47,0.44047619047619047
337227,35,26,0.5737704918032787
52087,34,27,0.5573770491803278
920568,30,43,0.410958904109589
875362,27,19,0.5869565217391305
558021,23,9,0.71875
155033,21,18,0.5384615384615384
265570,17,11,0.6071428571428571
912623,16,17,0.48484848484848486
942852,13,9,0.5909090909090909
1515819,13,12,0.52
350858,9,9,0.5
597384,9,8,0.5294117647058824
1015483,8,9,0.47058823529411764
718224,7,3,0.7
702469,7,3,0.7
